audio_service_web/lib/js/media_session_web.dart

```diff
@@ -1,39 +1,159 @@
-@JS('navigator.mediaSession')
+/// The interface to the Media Session API, `navigator.mediaSession`.
+///
+/// Useful links:
+///  * W3C specification https://w3c.github.io/mediasession/
+///  * MDN documentation https://developer.mozilla.org/en-US/docs/Web/API/MediaSession/
+@JS()
```

Comment:
I don't remember the full details of this now, but Flutter JS went through a transition period for a moment where some JS class mappings were built into Flutter and it was impossible to override them, hence the current hacky solution. Do we know the status of Flutter? Will this new solution work back to the minimum SDK version? (i.e. Flutter 2.0.0?)

Reply:
Found our previous discussion here: #574